addons/mail/views/res_config_settings_views.xml

```diff
@@ -135,6 +135,33 @@
                         </div>
                     </div>
                 </div>
+                <div id="companies_setting" position="inside">
+                    <br/>
+                    <div class="o_setting_right_pane" id="mail_templates_setting"
+                        groups="mail.group_mail_template_editor,base.group_system">
+                        <span class="o_form_label">Email Templates</span>
+                        <div class="text-muted">
+                            Customize the look and feel of automated emails
+                        </div>
+                        <div class="d-flex flex-row align-items-center">
```

Comment:
We generally try to avoid custom styling in form views to keep them as standard as possible. What is the purpose of that complicated styling ? Can't you use standard form classes like o_inline, ... ?